src/profile-logic/profile-data.js

```diff
@@ -1247,7 +1247,7 @@ export function convertStackToCallNodePath(
   ) {
     path.push(frameTable.func[stackTable.frame[stackIndex]]);
   }
-  return path;
+  return path.reverse();
```

Comment:
ooops this was incorrect here. I think that when I wrote that 1y ago I didn't quite understand these structures yet.